### PR TITLE
EPMDJ-4330: Fix console warning on Test2Code Page

### PR DESCRIPTION
**Pull request by SMPlay**
Issue: EPMDJ-4330

### CODE REVIEW

File: src/components/sort-arrows/sort-arrows.tsx [hunk 1]
```diff
@@ -13,19 +13,18 @@ interface Props {
...
-export const SortArrows = sortArrows(({
-  className, onClick, order,
-}: Props) => (
+export const SortArrows = sortArrows(({ className, onClick, order }: Props) => (
   <div className={className} onClick={onClick}>
-    <SortArrowWrapper>
-      <SortArrow active={order === 'ASC'} rotate={180} />
-      <SortArrow active={order === 'DESC'} />
-    </SortArrowWrapper>
+    <Wrapper>
```
Comment (RomanAchilov) [inline src/components/sort-arrows/sort-arrows.tsx:18]: SortArrowWrapper

File: src/components/sort-arrows/sort-arrows.tsx [hunk 2]
```diff
@@ -13,19 +13,18 @@ interface Props {
...
+      <Arrow active={order === 'ASC'}>
```
Comment (RomanAchilov) [inline src/components/sort-arrows/sort-arrows.tsx:19]: mb SortArrowUp and SortArrowDown

**Review by RomanAchilov — CHANGES_REQUESTED** [review]
you need to increasethe version in package.json

File: src/components/sort-arrows/sort-arrows.tsx (hunk 1 above)
Comment (SMPlay) [reply src/components/sort-arrows/sort-arrows.tsx:18]: Probably it's too much. I think this is clear from the context

**Review by RomanAchilov — APPROVED** [review]

**Review by RomanAchilov — CHANGES_REQUESTED** [review]
rename branch
rebase

**Pull request closed without merging**